docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -125,6 +134,73 @@ show up in ``DESCRIBE <table-name>`` or ``SELECT *``.
 
 This property is optional; the default is ``true``.
 
+``kafka.security-protocol``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Protocol used to communicate with brokers.
+Valid values are: PLAINTEXT, SSL.
+
+This property is optional; default is ``PLAINTEXT``.
+
+``kafka.ssl.keystore.type``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+File format of the keystore file.
+Valid values are: JKS, PKCS12.
+
+This property is optional, but required when ``kafka.ssl.keystore.location`` is given.
```

Comment:
Is this actually being enforced?

Reply:
I have removed this requirement based on Apache Kafka's documentation and [Confluent's documentation](https://docs.confluent.io/platform/current/kafka/authentication_ssl.html#id1) (note SSL configs examples omits setting up keystore.type/truststore.type). They both have this option marked as optional.


https://kafka.apache.org/documentation/#producerconfigs_ssl.keystore.type
> The file format of the key store file. This is optional for client.